src/sentry/migrations/0170_cool_users.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 1.11.29 on 2021-03-03 06:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0169_delete_organization_integration_from_projectcodeowners"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Handing in my homework late...
If we're removing a column we need to do so in several steps as described in the [docs](https://develop.sentry.dev/database-migrations/#deleting-columnstables)
Columns

1. Mark the column as nullable if it isn't, and create a migration.
2. Deploy.
3. Remove the column from the model, but in the migration make sure we only mark the state as removed.
4. Deploy.
5. Finally, create a migration that deletes the column.